facebook/delphiFacebook/R/variables.R

```diff
@@ -280,6 +280,40 @@ code_testing <- function(input_data) {
     input_data$t_tested_positive_14d <- NA
     input_data$t_wanted_test_14d <- NA
   }
+  
+  if ( "B10b" %in% names(input_data) ) {
+    testing_reasons <- split_options(input_data$B10b)
+    
+    input_data$t_tested_reason_sick <- is_selected(testing_reasons, "1")
+    input_data$t_tested_reason_contact <- is_selected(testing_reasons, "2")
+    input_data$t_tested_reason_medical <- is_selected(testing_reasons, "3")
+    input_data$t_tested_reason_employer <- is_selected(testing_reasons, "4")
+    input_data$t_tested_reason_large_event <- is_selected(testing_reasons, "5")
+    input_data$t_tested_reason_crowd <- is_selected(testing_reasons, "6")
+    input_data$t_tested_reason_visit_fam <- is_selected(testing_reasons, "7")
+    input_data$t_tested_reason_other <- is_selected(testing_reasons, "8")
+    
+    input_data$t_tested_reason_screening <- case_when(
+      input_data$t_tested_reason_sick == TRUE ~ 0,
+      input_data$t_tested_reason_contact == TRUE ~ 0,
+      input_data$t_tested_reason_crowd == TRUE ~ 0,
+      
+      input_data$t_tested_reason_medical == TRUE ~ 1,
```

Comment:
pretty sure that in 650, Taylor would have told you to do

```suggestion
      input_data$t_tested_reason_medical ~ 1,
```

because you don't need to compare against booleans like this. But I think it is actually more readable this way, for this specific case. If you agree, I'm fine with leaving it

Reply:
Good point. I've been using this format since some of our binary variables are coded as numeric rather than explicit logical and cause an error. I'm fine with either method.

Reply:
I think we can leave these for clarity, but set the `is.na` one below to use `!is.na()` instead. Somehow it reads better to me